assets/scripts/library-api.js

```diff
@@ -1,3 +1,35 @@
 'use strict';
 
-module.exports = true;
+const getBooksAuthor = function() {
+  return $.ajax({
+    url: 'http://localhost:3000/books/',
+    method: 'GET',
+  });
+};
+
+const getBooksTitle = function() {
+  return $.ajax({
+    url: 'http://localhost:3000/books/',
+    method: 'GET',
+  });
+};
+
+const getBooksId = function(id) {
+  return $.ajax({
+    url: 'http://localhost:3000/books/' + id,
+    method: 'GET',
+  });
+};
+
+const getAllBooks = function() {
```

Comment:
This would confuse future-me. The other functions begin with `getBooks*`, but this one starts with `getAll`. I'm sure I'd make a typo in a different file.